Gruntfile.js

```diff
@@ -413,6 +413,7 @@ module.exports = function (grunt) {
           'Attribute “ng-if” not allowed on element “p” at this point.',
           'Attribute “ng-if” not allowed on element “button” at this point.',
           'Attribute “ng-if” not allowed on element “img” at this point.',
+          'Attribute “ng-if” not allowed on element “hr” at this point.',
```

Comment:
The HTML linter is a pain, it doesn't allow custom attributes and custom tags and I can't figure out how to switch it off generically, so I end up ignoring the specific complaints...